QuickTCGA/Logic/TCGAUtilities.h

```diff
@@ -23,6 +23,8 @@
 
 #include <csignal>
 
+#include "vtkSlicerQuickTCGAModuleLogicExport.h"
```

Comment:
It is configured with the value specified here: https://github.com/SBU-BMI/SlicerPathology/blob/f07fbc8d7d7065b942bb3cae94a6aa4d36a1c3a9/QuickTCGA/Logic/CMakeLists.txt#L46